public/src/node.js

```diff
@@ -431,27 +432,34 @@ export default class Node {
     draw() {
         // console.log(this.id)
         const ctx = simulationArea.context;
+        const colors = getColors();
+        const color = colors["wire_draw"];
         if (this.clicked) {
             if (this.prev == 'x') {
-                drawLine(ctx, this.absX(), this.absY(), simulationArea.mouseX, this.absY(), 'black', 3);
-                drawLine(ctx, simulationArea.mouseX, this.absY(), simulationArea.mouseX, simulationArea.mouseY, 'black', 3);
+                drawLine(ctx, this.absX(), this.absY(), simulationArea.mouseX, this.absY(), color, 3);
+                drawLine(ctx, simulationArea.mouseX, this.absY(), simulationArea.mouseX, simulationArea.mouseY, color, 3);
             } else if (this.prev == 'y') {
-                drawLine(ctx, this.absX(), this.absY(), this.absX(), simulationArea.mouseY, 'black', 3);
-                drawLine(ctx, this.absX(), simulationArea.mouseY, simulationArea.mouseX, simulationArea.mouseY, 'black', 3);
+                drawLine(ctx, this.absX(), this.absY(), this.absX(), simulationArea.mouseY, color, 3);
+                drawLine(ctx, this.absX(), simulationArea.mouseY, simulationArea.mouseX, simulationArea.mouseY, color, 3);
             } else if (Math.abs(this.x + this.parent.x - simulationArea.mouseX) > Math.abs(this.y + this.parent.y - simulationArea.mouseY)) {
-                drawLine(ctx, this.absX(), this.absY(), simulationArea.mouseX, this.absY(), 'black', 3);
+                drawLine(ctx, this.absX(), this.absY(), simulationArea.mouseX, this.absY(), color, 3);
             } else {
-                drawLine(ctx, this.absX(), this.absY(), this.absX(), simulationArea.mouseY, 'black', 3);
+                drawLine(ctx, this.absX(), this.absY(), this.absX(), simulationArea.mouseY, color, 3);
             }
         }
-        var color = 'black';
-        if (this.bitWidth == 1) color = ['green', 'lightgreen'][this.value];
-        if (this.value == undefined) color = 'red';
+        let wire = colors['node_norm'];
+        const wireConnect = colors['wire_con']
+        const wirePow = colors['wire_pow']
+        const wireLose = colors['wire_lose']
+        const node = colors['node'];
+
+        if (this.bitWidth == 1) wire = [wireConnect, wirePow][this.value];
+        if (this.value == undefined) wire = wireLose;
         if (this.type == 2) this.checkHover();
-        if (this.type == 2) { drawCircle(ctx, this.absX(), this.absY(), 3, color); } else { drawCircle(ctx, this.absX(), this.absY(), 3, 'green'); }
```

Comment:
Something looks wrong here. Please use better names here. Names of color variables cannot be `wire` and `wireConnect`

Reply:
can I prefix them with something like 'color-wire', color-wireConnect'?

Reply:
can I prefix them with the word 'color'?